### ISSUE TITLE
save labels in json workflow

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-pyworkflow#172]
When I import a json workflow the labels are gone.

**Comment by pconesa** [comment]
Yep, labels are part of the "GUI" metadata and never was intended to be exported. But it would be nice. Thanks!

**Comment by azazellochg** [comment]
Also, save plugin binary versions and maybe description of each param.
